tests/emm/services/test_git_service.py

```diff
@@ -6,13 +6,21 @@
 
 import emm.services.git_service as under_test
 
+# TODO: Find a better way to handle the Path module when testing 3.7
```

Comment:
Can you open a ticket for this, if you haven't already?

Reply:
[DAG-1663](https://jira.mongodb.org/browse/DAG-1663)